specification/flag-evaluation/flag-evaluation.json

```diff
@@ -62,26 +62,26 @@
     {
         "id": "1.13",
         "clean id": "1_13",
-        "content": "The `evaluation details` structure's `variant` field SHOULD contain a identifier logically corresponding to the return flag value.",
-        "RFC 2119 keyword": "SHOULD"
+        "content": "In cases of normal execution, the `evaluation details` structure's `variant` field MUST contain the value of the `variant` field in the `flag resolution` structure returned by the configured `provider`.",
+        "RFC 2119 keyword": "MUST"
```

Comment:
can you remind me of the difference between the flag value (1.10) and the variant value here?

Reply:
This is like the name of the underlying value. Like you might have a giant json blob, but this is the ID or hash or descriptor of it.

Reply:
This is an optional field. I think this is another case where I was trying to simply say that the client should pass the value returned from provider, which may or may not be set. I will add clarification here.